core/plugins/stack/python/version.ts

```diff
@@ -40,6 +40,23 @@ file with a version inside, like "3.9".
 See https://github.com/pyenv/pyenv
 `;
 
+/*Search for a setup.py file with python version*/
+const setup: IntrospectFn<string | Error> = async (context) => {
+  for await (const file of context.files.each("**/setup.py")) {
+    const setupText = await context.files.readText(file.path);
+
+    const setupVersion: string | null = Array.from(
```

Comment:
The type `string | null` is already implied from the function return, it's not necessary to declare it here

```suggestion
    const setupVersion = Array.from(
```